src/go/rdctl/pkg/config/config.go

```diff
@@ -117,3 +127,36 @@ func finishConnectionSettings() (error, bool) {
 func insufficientConnectionInfo() bool {
 	return connectionSettings.Port == "" || connectionSettings.User == "" || connectionSettings.Password == ""
 }
+
+// determines if we are running in a wsl linux distro
+// by checking for availibily of wslpath and see if it's a symlink
+func isWSLDistro() bool {
+	fi, err := os.Lstat("/bin/wslpath")
+	if os.IsNotExist(err) {
+		return false
+	}
+	return fi.Mode()&os.ModeSymlink == os.ModeSymlink
+}
+
+func getAppDataPath() (string, error) {
+	subCommand := `chcp 65001 >nul & echo %APPDATA%`
+	cmd := exec.Command("cmd.exe", "/c", subCommand)
+	output, err := cmd.CombinedOutput()
```

Comment:
You only want to parse STDOUT; there may be errors on STDERR when you are running from a directory that doesn't exist on the Win32 side.